db/migrate/20221230055732_create_users_tasks_categories.rb

```diff
@@ -0,0 +1,22 @@
+class CreateUsersTasksCategories < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tasks do |t|
+      t.string :user_id # ユーザーID
+      t.string :name # タスク名
+      t.text :detail # タスク内容
+      t.time :start_date # タスク開始日
+      t.time :deadline # 期日
```

Comment:
time型は時間しか保存出来ないらしい。。datetime型が良さそう。
好みのレベルだけど、期日はend_dateにしてstart_dateと合わせるとスッキリしたイメージ。
```suggestion
      t.datetime :end_date # タスク終了日
```